include/logger/loggerApi.h

```diff
@@ -51,6 +51,7 @@ CPP_GUARD_BEGIN
         API_METHOD("getTrackCfg", api_getTrackConfig)                   \
         API_METHOD("getTrackDb", api_getTrackDb)                        \
         API_METHOD("getVer", api_getVersion)                            \
+        API_METHOD("get_wifi_ap_cfg", api_get_wifi_ap_cfg)              \
```

Comment:
:-1:   Wish you would have said that with the client API review.  But I don't much care so sure.